### ISSUE TITLE
Bars don't change when I choose specialization.

### ISSUE THREAD

**Issue by Maravelll** [tullamods/Dominos#877]
Dominos version: 11.0.1 -retail
Warcraft Retail. War Within.

Bars don't change when I choose specialization between Guardian and Balance. I reach lvl 80 and the configuration disappears. Don't jnow how to fix this.

![WoWScrnShot_082824_011409](https://github.com/user-attachments/assets/76cf1746-f54d-4ae0-a0a3-1d07728def06)
![WoWScrnShot_082824_011444](https://github.com/user-attachments/assets/1554d63b-d8fb-4b30-b3e8-ba02e222d320)

**Comment by Tuller** [comment]
Dominos doesn't actually save the contents of your bars on a per spec or profile basis.

**Comment by Maravelll** [comment]
Yes, it does. I use this addon since years. Is an amazing addon. Really improves the game experience. 

But dont mind. I found the problem. It seem that for some reason i dont know, during a dungeon call, after entering there, the spec changes automatically. The problem wasn't in this addons but in the Blizzard bar configuration, hosted in my "retail" or "cache" folders. i delete them and the problem was solved.

It seems to be a problem of the game itself.

So we can close this  thread.

Thanks for the help. Hope my solution could help anothers.
